client/wfnews-war/src/main/angular/src/app/components/public-event-page/evac-order-details/evac-order-details.component.ts

```diff
@@ -31,7 +31,7 @@ export class EvacOrderDetailsComponent {
       border: 'none'
     },
     clickHandler: () => {
-      window.open('https://www.emergencyinfobc.gov.bc.ca/');
+      window.open('https://www.emergencyinfobc.gov.bc.ca/', '_blank');
```

Comment:
thanks for addressing this, but might as well do the same to all other window.open (externalUrl) in this page